core/chat/chat.go

```diff
@@ -40,6 +40,11 @@ func GetClientsForUser(userID string) ([]*ChatClient, error) {
 	return clients[userID], nil
 }
 
+func FindClientById(clientId uint) (*ChatClient, bool) {
```

Comment:
If this isn't being used outside of the `chat` package you can un-export it (`findClientById`).

Reply:
Allright, made it private now.